src/models/branches/lines.jl

```diff
@@ -1,13 +1,41 @@
+function CheckThermalLimits(name::String,connectionpoints::@NT(from::Bus,to::Bus), r::Float64, x::Float64,rate::@NT(from_to::Float64, to_from::Float64),anglelimits::@NT(max::Float64, min::Float64))
```

Comment:
Is this actually doing any checking? We should rename to `calculate_thermal_limits`.

Reply:
This has been addressed with the latest commits 
bbf9a0f